make/modules/jdk.jpackage/Lib.gmk

```diff
@@ -67,10 +57,10 @@ $(eval $(call SetupJdkExecutable, BUILD_JPACKAGE_APPLAUNCHEREXE, \
     OPTIMIZATION := LOW, \
     DISABLED_WARNINGS_clang_LinuxPackage.c := format-nonliteral, \
     DISABLED_WARNINGS_clang_JvmLauncherLib.c := format-nonliteral, \
-    CXXFLAGS := $(call JpackageWithStaticCrt, $(CXXFLAGS_JDKEXE)) \
-        $(JPACKAGE_APPLAUNCHER_INCLUDES), \
-    CFLAGS := $(call JpackageWithStaticCrt, $(CFLAGS_JDKEXE)) \
-        $(JPACKAGE_APPLAUNCHER_INCLUDES), \
+    CFLAGS_FILTER_OUT := -MD, \
+    CXXFLAGS_FILTER_OUT := -MD, \
+    CXXFLAGS := $(JPACKAGE_APPLAUNCHER_INCLUDES), \
+    CFLAGS := $(JPACKAGE_APPLAUNCHER_INCLUDES), \
```

Comment:
This removed `-MD`, but I don't see it adding `-MT`.

Reply:
You are correct, this is missing in several places. 

I got curious on how this could still end up making no difference in the resulting binary, and I ended up in a deep google rabbit hole. My understanding is that `/MT` is in effect default, but Microsoft does not say that. Instead, they say that setting `/MT` makes the library link with `libcmt.lib` (and define `_MT`), but not setting any of `/MT` or `/MD` will result in your code linking with `libcmt.lib` by default... So if you don't look for the `_MT` define, there is no difference in the output. 

Nevertheless, we should have the option for clarity of what we mean.

Reply:
Actually, it **is** the default! I found it now: "/LD implies /MT unless you explicitly specify /MD." https://learn.microsoft.com/en-us/cpp/build/reference/md-mt-ld-use-run-time-library?view=msvc-170

Microsoft could certainly step up the game wrt their documentation. :-/